### PR TITLE
Removed the minimize and maximize buttons from the Properties window when using AppWindow

### PR DESCRIPTION
**Pull request by ahmed605**
**Resolved / Related Issues**
- None

**Details of Changes**
- Added `AppWindow` support for the file/folder Properties dialog
- Removed minimize and maximize buttons from the file/folder properties window when using `AppWindow`

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [x] Tested the changes for accessibility

**Screenshots (optional)**

https://user-images.githubusercontent.com/34550324/153066237-340f0730-57e6-499e-bd05-6fedb3869a9f.mp4

### CODE REVIEW

File: src/Files/Views/Pages/Properties.xaml [hunk 1]
```diff
@@ -38,6 +38,36 @@
...
+            VerticalAlignment="Top"
+            Background="Transparent">
+            <Rectangle
+                x:Name="CloseRect"
+                Width="46"
+                VerticalAlignment="Stretch"
+                Fill="Transparent"
+                PointerEntered="CloseRect_PointerEntered"
+                PointerExited="CloseRect_PointerExited"
+                PointerPressed="CloseRect_PointerPressed"
```
Comment (yaichenbaum) [inline src/Files/Views/Pages/Properties.xaml:55]: Can you add an automation name for screenreaders?

File: src/Files/Views/Pages/Properties.xaml.cs [hunk 2]
```diff
@@ -70,16 +83,84 @@ protected override void OnNavigatedTo(NavigationEventArgs e)
...
+
+                    crossIcon.Foreground = ThemeHelper.RootTheme switch
+                    {
+                        ElementTheme.Default => new SolidColorBrush((Color)Application.Current.Resources["SystemBaseHighColor"]),
+                        ElementTheme.Light => new SolidColorBrush(Colors.Black),
+                        ElementTheme.Dark => new SolidColorBrush(Colors.White),
+                        _ => new SolidColorBrush((Color)Application.Current.Resources["SystemBaseHighColor"])
+                    };
+
+                    AppWindowTitleBarCaptionButtonsGrid.Visibility = Visibility.Visible;
```
Comment (yaichenbaum) [inline src/Files/Views/Pages/Properties.xaml.cs:105]: Can you use conditional xaml in the xaml file instead of this?

Comment (ahmed605) [reply src/Files/Views/Pages/Properties.xaml.cs:105]: nope, we can't use custom functions/conditions in conditional XAML
We can bind the property but it doesn't worth it, that will also increase the complexity
If we really need to use binding I would suggest binding x:Load instead
What do you think?

File: src/Files/Views/Pages/Properties.xaml (hunk 1 above)
Comment (ahmed605) [reply src/Files/Views/Pages/Properties.xaml:55]: Sure, do we have a localization for the description of a close button?
Or we should just use "Close" (localized ofc)

Comment (yaichenbaum) [reply src/Files/Views/Pages/Properties.xaml:55]: You can reuse the localization that you used for the tooltip.

Comment (ahmed605) [reply src/Files/Views/Pages/Properties.xaml:55]: Done

**Review by yair100 — DISMISSED** [review]

File: src/Files/App.xaml.cs [hunk 3]
```diff
@@ -259,6 +259,8 @@ protected override async void OnLaunched(LaunchActivatedEventArgs e)
                     }
                 }
             }
+
+            WindowDecorationsHelper.RequestWindowDecorationsAccess();
```
Comment (gave92) [inline src/Files/App.xaml.cs:263]: To be put also in OnActivated and OnFileActivated?

Comment (yaichenbaum) [reply src/Files/App.xaml.cs:263]: I think it only needs to happen once in the app lifecycle.

**Comment by ahmed605** [comment]
> Windows 10, fully black in dark mode (if possible I'd keep the current color where mica is not suported):
> 
> 
> 
> Windows 10, with system in dark mode and app in light mode
> 
> 
> 

Hmmm this seems to be caused from @yair100 commits because it looked fine (normal Dark/Light colors) before these commits on my end (on Windows 10)

File: src/Files/App.xaml.cs (hunk 3 above)
Comment (ahmed605) [reply src/Files/App.xaml.cs:263]: Oh, so is OnActivated always called?
If so we can move that call to be only there

Comment (gave92) [reply src/Files/App.xaml.cs:263]: No I think we must put it in all 3 places. The method called depends how the app is launched.

**Review by yair100 — DISMISSED** [review]

**Review by gave92 — APPROVED** [review]

**Comment by yaichenbaum** [comment]
@ahmed605 thank you!

**Pull request merged**